pkg/api/convertertov20180930preview.go

```diff
@@ -20,10 +20,10 @@ func ConvertToV20180930preview(cs *OpenShiftManagedCluster) *v20180930preview.Op
 	}
 
 	oc.Plan = &v20180930preview.ResourcePurchasePlan{
```

Comment:
The internal ResourcePlan field probably has to change to a pointer too and avoid setting it here if cs.Plan == nil

Reply:
Changed ResourcePurchasePlan to a pointer, added checks in converters for nil. So that it's nil if admin/v20180903preview has a nil ResourcePurchasePlan.